Makefile

```diff
@@ -125,27 +125,6 @@ help: ## Display this help.
 
 ##@ Development
 
-DEFAULTS_TEST_FILE := test/support/defaults.go
-
-.PHONY: defaults
-defaults:
-	$(info Regenerating $(DEFAULTS_TEST_FILE))
-	@echo "package support" > $(DEFAULTS_TEST_FILE)
-	@echo "" >> $(DEFAULTS_TEST_FILE)
-	@echo "// ***********************" >> $(DEFAULTS_TEST_FILE)
-	@echo "//  DO NOT EDIT THIS FILE"  >> $(DEFAULTS_TEST_FILE)
-	@echo "// ***********************" >> $(DEFAULTS_TEST_FILE)
-	@echo "" >> $(DEFAULTS_TEST_FILE)
-	@echo "const (" >> $(DEFAULTS_TEST_FILE)
-	@echo "  CodeFlareSDKVersion = \"$(CODEFLARE_SDK_VERSION)\"" >> $(DEFAULTS_TEST_FILE)
-	@echo "  RayVersion = \"$(RAY_VERSION)\"" >> $(DEFAULTS_TEST_FILE)
-	@echo "  RayImage = \"$(RAY_IMAGE)\"" >> $(DEFAULTS_TEST_FILE)
```

Comment:
Can you also remove `CODEFLARE_SDK_VERSION`, `RAY_VERSION` and `RAY_IMAGE` references?
AFAIK they are used here, `CODEFLARE_SDK_VERSION` is also set in https://github.com/project-codeflare/codeflare-operator/blob/9912d96ccc33eec50061df248f4cfa4a6b4c934a/.github/workflows/tag-and-build.yml#L96

Reply:
That brings another point - should the codeflare-common repo be updated for every release (as it reference SDK version)?
IMHO SDK reference is just temporary, to be removed once https://github.com/project-codeflare/codeflare-sdk/issues/292 is implemented.

Reply:
Thanks. I have removed those references. 
Once that SDK PR is merged we can remove the references to its version in codeflare-common.  I can create a follow on issue for this. 
The issue is blocked at the moment so I it should probably be updated for any releases in the meantime?

Reply:
Right

Reply:
Follow up issue: https://github.com/project-codeflare/codeflare-common/issues/5 
What do we need to do to ensure it is updated in the meantime?

Reply:
I am thinking whether it has sense to compose the codeflare-common update into release workflow...
That would be ideal solution but quite a big work considering that it would be removed soon. I would say that the reference can be updated manually in meantime until the SDK tests are implemented.

Reply:
Ok. Do we need to create an issue or track this anywhere?

Reply:
Yes, issue will be needed.
I will create it.

Reply:
Created - https://github.com/project-codeflare/codeflare-common/issues/6